lib/dl_sqlalchemy_chyt/dl_sqlalchemy_chyt/table_engine.py

```diff
@@ -0,0 +1,14 @@
+from typing import Any
+
+from clickhouse_sqlalchemy.engines import Engine
+
+
+class YtTable(Engine):
+    def __init__(self, **kwargs: Any):
+        self.engine_kwargs = kwargs
+        super().__init__()
+
+    def get_parameters(self) -> str:  # Method should return something stringable
+        if not self.engine_kwargs:
+            return ""
+        return f"{{{','.join([f'{key}={value}' for key, value in sorted(self.engine_kwargs.items())])}}}"
```

Comment:
nitpick: please split this in multiple lines, `))])}}}"` physically hurts to read